sys/dev/amr/amr_pci.c

```diff
@@ -116,11 +116,6 @@ static driver_t amr_pci_driver = {
 	sizeof(struct amr_softc)
 };
 
-static devclass_t	amr_devclass;
-DRIVER_MODULE(amr, pci, amr_pci_driver, amr_devclass, 0, 0);
-MODULE_DEPEND(amr, pci, 1, 1, 1);
-MODULE_DEPEND(amr, cam, 1, 1, 1);
-
 static struct amr_ident
 {
     int		vendor;
```

Comment:
but we need to change vendor and device here to be uint16_t's rather than int's. int's are 32-bits, but a poor fit for this data.